fingerprinting/find_docker_sock.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 # look for the docker socket if it was (stupidly) mounted.
-find "/" 2>&1 | grep -E "(.*\/docker\.sock|^docker\.sock)$"
\ No newline at end of file
+find "/" -name docker.sock 2>&1  
```

Comment:
```suggestion
find "/" -name docker.sock 2>/dev/null
```
Changing `2>&1` to `/dev/null`